openreview/openreview.py

```diff
@@ -1043,7 +1044,7 @@ def delete_group(self, group_id):
         return response.json()
 
 
-    def post_message(self, subject, recipients, message):
+    def post_message(self, subject, recipients, message, ignoreRecipients=None, sender=None):
```

Comment:
Is this param `sender` implemented in the backend ?

Reply:
https://github.com/openreview/openreview/pull/1932

Reply:
https://github.com/openreview/openreview/pull/1926

Reply:
these are the two PRs already merged in master, I've been using this to send the reminders.